### PR TITLE
Editing README file

### PR DESCRIPTION
**Pull request by SreeVedantham**
Added Sample Text

### CODE REVIEW

**Comment by kyerramilly** [comment]
👍 

@SreeVedantham 
Please remove the empty lines of code

**Comment by SreeVedantham** [comment]
@kyerramilly 
I have now removed empty lines in the code.

File: README.md [hunk 1]
```diff
@@ -1,2 +1,3 @@
 # Sample
 Sample
+Sample Text
```
Comment (kyerramilly) [inline README.md]: @SreeVedantham 
Instead of Sample Text please use Lorem Ipsum..... Place holder text

Comment (SreeVedantham) [reply README.md]: @kyerramilly 

Updated using lorem ipsum

**Comment by kyerramilly** [comment]
👍

**Pull request merged**